packages/js/test/plugins/auctionHouseModule/directBuy.test.ts

```diff
@@ -0,0 +1,147 @@
+import test, { Test } from 'tape';
+import spok, { Specifications } from 'spok';
+import { sol, token } from '@/types';
+import {
+  assertThrows,
+  createNft,
+  createWallet,
+  killStuckProcess,
+  metaplex,
+  spokSameAmount,
+  spokSamePubkey,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+import { findAssociatedTokenAccountPda, Purchase } from '@/index';
+import { Keypair } from '@solana/web3.js';
+
+killStuckProcess();
+
+test('[auctionHouseModule] buy on an Auction House with minimum input', async (t: Test) => {
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+  const buyer = await createWallet(mx);
+
+  const nft = await createNft(mx);
+  const auctionHouse = await createAuctionHouse(mx);
+
+  // And we listed that NFT for 1 SOL.
+  const { listing } = await mx
+    .auctionHouse()
+    .list({
+      auctionHouse,
+      mintAccount: nft.address,
+      price: sol(1),
+    })
+    .run();
+
+  // When we execute direct buy with the given listing.
+  const { purchase } = await mx
+    .auctionHouse()
+    .buy({
+      auctionHouse,
+      listing,
+      buyer,
```

Comment:
"Minimum inputs" means we should not provide the `buyer` which defaults to the identity. Instead, the `seller` should be explicitly defined so they are different wallets.

Reply:
@zaxozhu Don't forget about this. The buyer should not be passed in if we want minimum input.